.github/workflows/release.yml

```diff
@@ -65,8 +64,11 @@ jobs:
         include:
           - platform: x86_64
             base_image: ubuntu:22.04
+            runs-on: ubuntu-latest
           - platform: arm64
-            base_image: arm64v8/ubuntu:22.04
+            base_image: ubuntu:22.04
+            runs-on: ubuntu-24.04-arm
```

Comment:
only `ubuntu-22.04-arm` or `ubuntu-24.04-arm` is available.
https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/#how-to-use-the-runners